test/WebSites/MvcTagHelpersWebSite/wwwroot/site.js

```diff
@@ -0,0 +1 @@
+﻿
```

Comment:
You can see if it loads just fine with the browser tools :smile: 
I do the alert in the other js files because they should never ever load.